prometheus-stats

```diff
@@ -45,40 +42,67 @@ def main():
     db_conn = sqlite3.connect(opts.db)
     cursor = db_conn.cursor()
 
-    since = time.time() - opts.hours * HOUR
-
     # Output in prometheus format, see:
     # https://prometheus.io/docs/instrumenting/exposition_formats/
     output = ""
 
-    # Get total number of runs and wait time sum in last N hours
+    # Get total number of runs and wait time sum in last 24 hours
     (test_runs, test_runs_queue_wait_sum) = cursor.execute("""\
             SELECT COUNT(*), SUM(wait_seconds)
             FROM TestRuns
-            WHERE time > ? ;""", (since, )).fetchone()
+            WHERE time > strftime('%s', 'now', '-24 hours')""").fetchone()
```

Comment:
Note to self: I think we've been approaching this wrong all along. We should move from gauges to counters, and start from the beginning of (available) history. Then select the time span in Prometheus/Grafana, with either something like `test_runs_queue_wait*[24h]` or using functions like `rate()`. Prometheus knows how to deal with counter resets, and that way we don't have to choose this "look-back window" in advance. I'll look at that in a follow-up.